purchases/src/main/kotlin/com/revenuecat/purchases/common/events/BackendEvent.kt

```diff
@@ -0,0 +1,51 @@
+package com.revenuecat.purchases.common.events
+
+import com.revenuecat.purchases.utils.Event
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+internal sealed class BackendEvent : Event {
+    @Serializable
+    data class CustomerCenter(
+        val id: String,
+        @SerialName("revision_id")
+        val revisionID: Int,
+        val type: String,
+        @SerialName("app_user_id")
+        val appUserID: String,
+        @SerialName("session_id")
+        val appSessionID: String,
+        val timestamp: Long,
+        @SerialName("dark_mode")
+        val darkMode: Boolean,
+        val locale: String,
+        @SerialName("is_sandbox")
+        val isSandbox: Boolean,
+        @SerialName("display_mode")
+        val displayMode: String,
+    ) : BackendEvent()
+
+    @Serializable
+    @SerialName("paywalls")
+    data class Paywalls(
+        val id: String,
+        val version: Int,
+        val type: String,
+        @SerialName("app_user_id")
+        val appUserID: String,
+        @SerialName("session_id")
+        val sessionID: String,
+        @SerialName("offering_id")
+        val offeringID: String,
+        @SerialName("paywall_revision")
+        val paywallRevision: Int,
+        val timestamp: Long,
+        @SerialName("display_mode")
+        val displayMode: String,
+        @SerialName("dark_mode")
+        val darkMode: Boolean,
+        @SerialName("locale")
+        val localeIdentifier: String,
```

Comment:
Looking at iOS, it looks like this should be `locale`? I might be wrong, but it would be good to double check

Reply:
Yes, it's locale for the event we send to the SDK (see EventsRequest+Paywall.swift)

```swift
    private enum CodingKeys: String, CodingKey {

        case id
        case version
        case type
        case appUserID = "appUserId"
        case sessionID = "sessionId"
        case offeringID = "offeringId"
        case paywallRevision
        case timestamp
        case displayMode
        case darkMode
        case localeIdentifier = "locale"

    }
```

Reply:
We're good, it's locale in iOS as well